<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tcp with Qt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="2">
          <widget class="QPushButton" name="pushBSend">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" rowspan="3">
          <widget class="QLineEdit" name="lineEPort"/>
         </item>
         <item row="5" column="0" rowspan="2">
          <widget class="QLabel" name="labMsg">
           <property name="text">
            <string>发送数据</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="3">
          <widget class="QLineEdit" name="lineEIP"/>
         </item>
         <item row="1" column="2" rowspan="4">
          <widget class="QPushButton" name="pushBConnect">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labPort">
           <property name="text">
            <string>服务器端口</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEMsg"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labIP">
           <property name="text">
            <string>服务器IP</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditMsg"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
